paasta_tools/cli/cmds/metastatus.py

```diff
@@ -121,15 +125,19 @@ def paasta_metastatus(args):
     system_paasta_config = load_system_paasta_config()
     all_clusters = list_clusters(soa_dir=soa_dir)
     clusters_to_inspect = figure_out_clusters_to_inspect(args, all_clusters)
+    return_codes = []
     for cluster in clusters_to_inspect:
         if cluster in all_clusters:
-            print_cluster_status(
-                cluster=cluster,
-                system_paasta_config=system_paasta_config,
-                humanize=args.humanize,
-                groupings=args.groupings,
-                verbose=args.verbose
+            return_codes.append(
+                print_cluster_status(
+                    cluster=cluster,
+                    system_paasta_config=system_paasta_config,
+                    humanize=args.humanize,
+                    groupings=args.groupings,
+                    verbose=args.verbose,
+                )
             )
         else:
             print "Cluster %s doesn't look like a valid cluster?" % args.clusters
             print "Try using tab completion to help complete the cluster name"
+    return all([return_code == 0 for return_code in return_codes])
```

Comment:
Is this doing what you expect? Won't it return True if everything returns 0 and then end up with `sys.exit(True)`?